Elemental/Components/AeTable/AePaginatedTable.razor

```diff
@@ -88,6 +88,7 @@
     protected override async Task OnParametersSetAsync()
     {
         LoadPageItems();
+        Total = Rows.Count();
```

Comment:
Nice.

I think there was some discussion about having the table be like a sliding window over the items. Where you'd hook into asynchronous functions to fetch each page's items and the parent component would handle setting the total.

We'll have to bring it up in a planning meeting to see if that's somewhere we want to take it.

Reply:
Ok @Minhja101  I'm going to make a decision on this. This component will not be asynchronously loading the paging elements. We will handle that in a future component or revisit this.

This fixes a bug when new data is passed in so we can remove the `Total = Rows.Count();` from the `OnInitialized` method.